.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+      {
+          "name": "Launch",
```

Comment:
This `launch.json` is not really useful since we now have 3 binaries. Therefore, we should have
- Launch nv2
- Launch docker-generate
- Launch docker-nv2